packages/now-next/tsconfig.json

```diff
@@ -7,7 +7,9 @@
     "module": "commonjs",
     "outDir": "dist",
     "sourceMap": false,
-    "declaration": false
+    "declaration": false,
+    "allowSyntheticDefaultImports": true,
```

Comment:
I don't understand why `allowSyntheticDefaultImports` was enabled. That should be the same as `esModuleInterop` which is already enabled.

Reply:
Yea, I've ran into some issues and thought that would do it, but I'll remove it again